tests/qml/Properties.qml

```diff
@@ -12,5 +12,5 @@ Rectangle {
   property int number2: 10;
   property var foo: {}
   property var bar: []
-  property Item item: Item {}
+  property Item item: QQ.Item {}
```

Comment:
This should be added as a new line, and should not replace the current `property Item item: Item {}` test.